packages/nextdoor-token-refresher/index.ts

```diff
@@ -0,0 +1,101 @@
+import { getParameter, getParametersByPath, putParameter } from "@create-disruptions-data/shared-ts/utils/ssm";
+import { nextdoorTokenSchema } from "@create-disruptions-data/shared-ts/utils/zod";
+import * as logger from "lambda-log";
+import { randomUUID } from "crypto";
+
+export const nextdoorRedirectUri = `${process.env.DOMAIN_NAME as string}/api/nextdoor-callback`;
+
+export const getNextdoorClientIdAndSecret = async () => {
+    const [nextdoorClientIdKeyParam, nextdoorClientSecretParam] = await Promise.all([
+        getParameter("/social/nextdoor/client_id", logger),
+        getParameter("/social/nextdoor/client_secret", logger),
+    ]);
+
+    const nextdoorClientId = nextdoorClientIdKeyParam.Parameter?.Value ?? "";
+    const nextdoorClientSecret = nextdoorClientSecretParam.Parameter?.Value ?? "";
+
+    return { nextdoorClientId, nextdoorClientSecret };
+};
+
+export const getNextdoorAuthHeader = async () => {
+    const { nextdoorClientId, nextdoorClientSecret } = await getNextdoorClientIdAndSecret();
+    if (!nextdoorClientId || !nextdoorClientSecret) {
+        return "";
+    }
+    const key = `${nextdoorClientId}:${nextdoorClientSecret}`;
+
+    return `Basic ${Buffer.from(key).toString("base64")}`;
+};
```

Comment:
This code is duplicated in `nextdoor.ts` - can we move it to the shared-ts folder please?

Reply:
done